webui-src/app/people/people_ownids.js

```diff
@@ -1,213 +1,379 @@
+                            id = owns.ids[0];
+                            console.log(id);
+                          }
+                          rs.rsJsonApiRequest(
+                            "/rsIdentity/createIdentity",
+                            {
+                              id: id,
+                              name: name,
+                              //avatar: v.attrs.details.mAvatar.mData.base64,
+                              pseudonimous: pseudonimous,
+                              pgpPassword: paraphase,
+                            },
+                            (data) => {
+                              const message = data.retval
+                                ? "Successfully created identity."
+                                : "An error occured while creating identity.";
+                              widget.popupMessage([
+                                m("h3", "Create new Identity"),
+                                m("hr"),
+                                message,
+                              ]);
+                            }
+                          );
+                        }
+                      );
+                    },
+                  },
+                  "Enter"
+                ),
+              ]);
+            } else
+              rs.rsJsonApiRequest(
+                "/rsIdentity/createIdentity",
+                {
+                  name,
+                  pseudonimous,
+                  pgpPassword,
+                },
+                (data) => {
+                  const message = data.retval
+                    ? "Successfully created identity."
+                    : "An error occured while creating identity.";
+                  widget.popupMessage([
+                    m("h3", "Create new Identity"),
+                    m("hr"),
+                    message,
+                  ]);
+                }
+              );
+          },
         },
-        'Create',
+        "Create"
       ),
     ],
   };
 };
 
 const EditIdentity = () => {
+  let name = "";
   return {
-    view: () => [
-      m('i.fas.fa-user-edit'),
-      m('h3', 'Edit Identity'),
-      m('hr'),
-      m('input[type=text][placeholder=Name]', {}),
-      m('canvas'),
-      m('button', 'Save'),
+    view: (v) => [
+      m("i.fas.fa-user-edit"),
+      m("h3", "Edit Identity"),
+      m("hr"),
+      m("input[type=text][placeholder=Name]", {
+        value: name,
+        oninput: (e) => {
+          name = e.target.value;
+        },
+      }),
+      m("canvas"),
+      m(
+        "button",
+        {
+          onclick: () => {
+            if (!checksudo(v.attrs.details.mPgpId)) {
+              console.log(v.attrs.details.mPgpId);
+              let paraphase = "";
+              widget.popupMessage([
+                m("i.fas.fa-user-edit"),
+                m("h3", "Enter your passpharse"),
+                m("hr"),
+
+                m("input[type=password][placeholder=Passpharse]", {
```

Comment:
Again, better to create a separate component here. Since this input is the only difference between signed and unsigned, we can enable it with optional prop.